aptos-node/src/storage.rs

```diff
@@ -30,49 +30,44 @@ pub(crate) fn maybe_apply_genesis(
 }
 
 #[cfg(not(feature = "consensus-only-perf-test"))]
-pub(crate) fn bootstrap_db(
```

Comment:
nit: can we not override this entire function? We'll lose the blame commit this way.